<launch>
  <!-- vim: set ft=xml noet : -->
  <!-- base node launch file-->

  <arg name="my_id" default="1" />
  <arg name="flag_1vrpn_2vio_3both" default="3" />
  <arg name="vio_odomTopic" default="/camera/odom/sample" />

  <node pkg="vision_pose" type="vision_pose" name="vision_pose" output="screen">
    <!-- Param -->
    <param name="my_id" value="$(arg my_id)" type="int" />
    <param name="flag_1vrpn_2vio_3both" value="$(arg flag_1vrpn_2vio_3both)" type="int" />
    <param name="vio_odomTopic" value="$(arg vio_odomTopic)" type="string" />
  </node>

</launch>
